frontend_app_workspace_advanced/src/component/SpaceDescription.jsx

```diff
@@ -93,7 +94,7 @@ export const SpaceDescription = (props) => {
         : (
           <div className='workspace_advanced__description'>
             {props.description
-              ? <div dangerouslySetInnerHTML={{ __html: props.description }} />
+              ? <HTMLContent showImageBorder={false}>{ props.description }</HTMLContent>
```

Comment:
Please add a comment explaining that some users use images for advanced description layout so we must not add borders